CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
-## [2.0] - 2017-05-24
+## Unreleased
+
+### Fixed
+- Update `webpack-dev-server.tt` to respect RAILS_ENV and NODE_ENV values [#502](https://github.com/rails/webpacker/issues/502)
```

Comment:
Loosely. It was a necessary change along with this PR for the ability to run the webpack-dev-server in test. Would you prefer a separate PR for this entry?